cosmwasm/utils.js

```diff
@@ -451,7 +451,7 @@ const makeAxelarnetGatewayInstantiateMsg = (config, chainName) => {
 
     return {
         router_address: routerAddress,
-        nexus_gateway: nexusAddress,
+        nexus: nexusAddress,
```

Comment:
this is the nexus module account now

Reply:
Nexus gateway shouldn't be deployed anymore. And core `gateway` param should point to `axelarnet-gateway` address